doc/theoretical_background.rst

```diff
@@ -7,158 +7,112 @@ Theoretical Background
-certain parameter like for example the distances between old coordinates and
-their nearest neighbor in the set of new coordinates. The method was
-implemented by `Hoersch et al. <https://arxiv.org/pdf/1705.07617.pdf>`_ within
-PyPSA.
-
-Snapshot skipping
-^^^^^^^^^^^^^^^^^
-This method simplifies the simulation temporally by considering every n-th
-snapshot of a given time series. The regarded snapshots are weighted by the
-number of neglected snapshots to ensure a comparable calculation of costs.
-This method assumes the chosen snapshots to be represenative for the next n-th
-snapshots in the time series.
-
-Snapshot-Clustering
-^^^^^^^^^^^^^^^^^^^
-This method aggregate given time series for various time intervals like i.e.
-days using the `tsam <https://pypi.org/project/tsam>`_ package. Contrary to
-snapshot skipping, this approach averages a certain period of snapshots
-instead of choosing a representative snapshot.
-
-
-Storage expansion
------------------
-To evaluate the amount of storage units in future energy systems, the possible
-installation of new storage units at every node in the network is allowed. The
-size and operation of these storages are part of the optimization problem.
-
-Two types of storage technologies are considered - batteries and hydrogen in
-underground caverns. Li-Ion battery storages as representatives for short-term
-(several hours) technologies, which can be installed at every node.
-Underground hydrogen storages represent long-term or seasonal (weeks)
-technologies and can be build at every node with appropriate salt formations
-in the underground. The storage parameters for both types are reached by
-[Acatech2015]_, the information about salt formations are given by [BGR]_.
-
-
-Grid expansion
---------------
-The grid expansion is realized by extending the capacities of existing
-lines and substations. These capacities are regarded as part of the
-optimization problem, whereby the possible extension is unlimited. With respect
-to the different voltage levels and lengths MVA-specific costs are considered
-in the linear optimization of the power flow. Besides, several planned grid
-expansion scenarios from the German grid development plan can be considered as
-possible additional power lines by using the 'scn_extension' argument.
+=======
+
+
+Optimization with PyPSA
+-----------------------
+
+Within eTraGo, the fetched data model is translated into a `PyPSA <https://pypsa.readthedocs.io/en/latest/>`_-network. The power flow simulations and optimizations are performed with a linear approximation assuming eTraGo to fulfill the assumptions to perfom a LOPF (as those are small voltage angle differences, branch resistances negligible to their reactances, voltage magnitudes can be kept at nominal values) since it focuses on the extra-high and high voltage levels. As objective value of the optimization, the overall system costs are considered.
```

Comment:
Here you write that pf simulation and LOPF optimization is linear. But the power flow is non-linear, isn't it? That is also written in the next sentence, so it might be easier to only write about the optimization in this sentences.